tests/test_backend.py

```diff
@@ -2932,6 +2933,27 @@ def func(input_holder):
         for input_val in input_vals:
             self._run_test_case(func, [_OUTPUT], {_INPUT: input_val})
 
+    @skip_tf_cpu("only tf_gpu can run CudnnGPU")
```

Comment:
Those tests are significant different compared to what is in test_backend.py that they should be in their own file, ie. test_cudnn.py ?